src/elements/dialog/dialog-content/dialog-content.ts

```diff
@@ -16,7 +18,7 @@ class SbbDialogContentElement extends LitElement {
 
   protected override render(): TemplateResult {
     return html`
-      <div class="sbb-dialog-content">
+      <div class="sbb-dialog-content" @scroll=${(e: Event) => forwardEvent(e, document)}>
         <slot></slot>
       </div>
```

Comment:
Out of curiosity, if we don't forward the scroll event the `capture` does not intercept it?

Reply:
no, because the scroll event is not composed, so it stays inside shadow DOM, even when using capture.